interface/src/gmap.test.js

```diff
@@ -0,0 +1,53 @@
+import React from "react"
+import { shallow } from 'enzyme'
+import GMap from './components/gmap'
+
+describe('Test the GMap Component',()=>{
+    it('it should render correctly',()=>{
+        const gmap = shallow(<GMap />, {
+            disableLifecycleMethods: true
+        });
+        const geoJSON= {
```

Comment:
A nice enhancement here would be to have the JSON loaded from file as in gmap.js. This would allow the data that's checked against to be easily changed.

Reply:
The visualization server parses the shapefile and then returns it to the frontend. So for testing the UI, the interface will be dependent on another component which can also be done by providing sample data. If somehow, the visualization part is not working as expected in the future, we can still test the interface on the provided data.